src/cloud/components/Views/index.tsx

```diff
@@ -128,11 +147,13 @@ export const ViewsManager = ({
 
   const selectViewId = useCallback(
     (id: number) => {
+      replace(`?view=${id}`)
```

Comment:
Is it intended? Is there any reasons to use replace instead of push? I think we should use push so users can switch views by `back` and `forward` button of their browser.